schemas/src/definitions/inputKind.yaml

```diff
@@ -4,22 +4,35 @@ $id:     <HOST>/<PREFIX>/<VERSION>/definitions/inputKind.yaml
 
 title: Executable Command Input Type
 description: >-
-  Defines how DSC should pass input to the command, either as arguments or
-  JSON over stdin.
+  Defines how DSC should pass input to the command, either as environment variables or
+  JSON over stdin. When this value isn't defined, DSC doesn't send the resource any input.
 type: string
 enum:
-  - args
+  - env
   - stdin
-default: args
 
 # VS Code only
-markdownDescription: |
-  Defines how DSC should pass input to the command, either as arguments or JSON over `stdin`.
+# Don't define markdownDescription - each schema using this must define it, to link correctly.
 markdownEnumDescriptions:
-  # args
-  - |
-      Indicates that the resource expects the properties of an instance to be specified with
-      command line arguments. This option isn't implemented yet.
-  # stdin
-  - |
-      Indicates that the resource expects a JSON blob representing an instance from `stdin`.
+  - | # env
+      _Environment variables_
+
+      > Indicates that the resource expects the properties of an instance to be specified as
+      > environment variables. This option only supports the following data types for instance
```

Comment:
Should we be more explicit that JSON property names become the name of the env var (case preserving) and the value is as-is.  Arrays become comma delimited strings.